### PR TITLE
Fix documentation

### PR DESCRIPTION
**Pull request by bijayprasadtiwari**
Added samples for view and protection job, added md files for newly added cmdlets.

### CODE REVIEW

File: docs/cmdlets-reference/get-cohesityvault.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Get-CohesityVault -VaultName <string> [<CommonParameters>]
+```
+
+## DESCRIPTION
+List the Vaults (External Targets) registered on the Cohesity Cluster filtered by the specified parameters.
+
+## EXAMPLES
+
+### EXAMPLE 1
+Get-CohesityVault -VaultName <string>
```
Comment (ashish-cohesity) [inline docs/cmdlets-reference/get-cohesityvault.md]: Should be actual example

Comment (bijayprasadtiwari) [reply docs/cmdlets-reference/get-cohesityvault.md]: Fixed

File: samples/utility/Backup-VMAttributes.ps1 [hunk 2]
```diff
@@ -0,0 +1,91 @@
+<#
```
Comment (ashish-cohesity) [inline samples/utility/Backup-VMAttributes.ps1]: Should be in protectionjob folder as well, and also remove the vcenterIPs, put a fqdn. This and script below.

Comment (bijayprasadtiwari) [reply samples/utility/Backup-VMAttributes.ps1]: The scripts do not use cohesity cmdlet, does not make sense to place it inside protectionJob folder

**Pull request merged**